src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star size"></div>
+      <div class="stars__star size"></div>
+      <div class="stars__star size"></div>
+      <div class="stars__star size"></div>
+      <div class="stars__star size"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active size"></div>
```

Comment:
you don't need this extra modifiers, just use nth-child pseudo-class for setting bg-image